tests/unit/cli/admin/test_import_targets.py

```diff
@@ -0,0 +1,541 @@
+                    )
+                )
+            )
+        )
+
+        result = import_targets._get_succinct_roles(
+            "http://127.0.0.1/metadata"
+        )
+        assert result == "fake_succinct_roles"
+
+    def test__get_succinct_roles_not_found_metadata(self, monkeypatch):
+        import_targets.request_server = pretend.call_recorder(
+            lambda *a: pretend.stub(status_code=404, text=b"data")
+        )
+
+        with pytest.raises(import_targets.click.ClickException) as err:
+            import_targets._get_succinct_roles("http://127.0.0.1/metadata")
+        assert "RSTUF Metadata Targets not found." in str(err)
+
+    def test__get_succinct_roles_no_delegations(self, monkeypatch):
+        import_targets.request_server = pretend.call_recorder(
+            lambda *a: pretend.stub(status_code=200, text=b"data")
+        )
+        monkeypatch.setattr(
+            import_targets.json, "loads", lambda *a: "json_data"
+        )
+        import_targets.Metadata.from_dict = pretend.call_recorder(
+            lambda *a: pretend.stub(signed=pretend.stub(delegations=None))
+        )
+
+        with pytest.raises(import_targets.click.ClickException) as err:
+            import_targets._get_succinct_roles("http://127.0.0.1/metadata")
+        assert "Failed to get Targets Delegations" in str(err)
+
+    def test__get_succinct_roles_no_succinct_roles(self, monkeypatch):
+        import_targets.request_server = pretend.call_recorder(
+            lambda *a: pretend.stub(status_code=200, text=b"data")
+        )
+        monkeypatch.setattr(
+            import_targets.json, "loads", lambda *a: "json_data"
+        )
+        import_targets.Metadata.from_dict = pretend.call_recorder(
+            lambda *a: pretend.stub(
+                signed=pretend.stub(
+                    delegations=pretend.stub(succinct_roles=None)
+                )
+            )
+        )
+
+        with pytest.raises(import_targets.click.ClickException) as err:
+            import_targets._get_succinct_roles("http://127.0.0.1/metadata")
+        assert "Failed to get Targets succinct roles" in str(err)
+
+
+class TestImportTargetsGroupCLI:
+    def test_import_targets(self, client, test_context):
+        test_context["settings"].SERVER = "fake-server"
+        test_context["settings"].TOKEN = "test-token"
+
+        import_targets.is_logged = pretend.call_recorder(
+            lambda *a: pretend.stub(state=True)
+        )
+        import_targets.get_headers = pretend.call_recorder(
+            lambda *a: "headers"
+        )
+        fake_response = pretend.stub(
+            status_code=200,
+            json=pretend.call_recorder(lambda: {"data": {"bootstrap": True}}),
+        )
+        import_targets.request_server = pretend.call_recorder(
+            lambda *a, **kw: fake_response
+        )
+        import_targets._get_succinct_roles = pretend.call_recorder(
+            lambda *a: "fake_succinct_roles"
+        )
```

Comment:
You forgot to assert the calls for `import_targets._get_succinct_roles` after invoking `import_targets._get_succinct_roles`.